docs/SUMMARY.md

```diff
@@ -1,14 +1,13 @@
 # Summary
 
 -   [Introduction](index.md)
-
     -   [Analytics](analytics/index.md)
     -   [AWS](./analytics/aws.md)
     -   [Sentry](./analytics/sentry.md)
+    <!-- todo: looks like links outside docs folder don't work -->
```

Comment:
I think this could be solved by an absolute URL. However, I am not sure if it is a good practice to link the readmes from docs. They mostly contain info not relevant to users or 3rd party developers. If they do, we should move this info to docs.

Reply:
cc @tomasklim